benchmark/single-source/SetTests.swift

```diff
@@ -135,10 +211,133 @@ public let SetTests = [
+    name: "Set.isStrictSubset.Seq.Empty.Int",
+    runFunction: { n in run_SetIsStrictSubsetSeqInt(setE, arrayAB, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Seq.Int.Empty",
+    runFunction: { n in run_SetIsStrictSubsetSeqInt(setAB, arrayE, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayE]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Seq.Int0",
+    runFunction: { n in run_SetIsStrictSubsetSeqInt(setAB, arrayCD, false, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayCD]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Seq.Box0",
+    runFunction: { n in run_SetIsStrictSubsetSeqBox(setOAB, arrayOCD, false, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, arrayOCD]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Seq.Int25",
+    runFunction: { n in run_SetIsStrictSubsetSeqInt(setB, arrayAB, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setB, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Seq.Box25",
+    runFunction: { n in run_SetIsStrictSubsetSeqBox(setOB, arrayOAB, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOB, arrayOAB]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Seq.Int50",
+    runFunction: { n in run_SetIsStrictSubsetSeqInt(setY, arrayXY, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setY, arrayXY]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Seq.Int100",
+    runFunction: { n in run_SetIsStrictSubsetSeqInt(setP, arrayP, false, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setP, arrayP]) }),
+
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Empty.Int",
+    runFunction: { n in run_SetIsSupersetSeqInt(setAB, arrayE, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayE]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Int.Empty",
+    runFunction: { n in run_SetIsSupersetSeqInt(setE, arrayAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Int0",
+    runFunction: { n in run_SetIsSupersetSeqInt(setCD, arrayAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setCD, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Box0",
+    runFunction: { n in run_SetIsSupersetSeqBox(setOCD, arrayOAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOCD, arrayOAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Int25",
+    runFunction: { n in run_SetIsSupersetSeqInt(setAB, arrayB, true, 50 * n) },
```

Comment:
```suggestion
    runFunction: { n in run_SetIsSupersetSeqInt(setB, arrayBC, false, 50 * n) },
```
* Analogously for `Box25`.
* Use `setY` and `arrayYZ` on `Int50` below.
* Adjust `setUpFunction`s to match.

Reply:
Trying to hit early exit at line 707 in Set. (This is basically unmodified by #21300.)